include/glow/Graph/Context.h

```diff
@@ -65,8 +63,17 @@ class Context final {
   /// found.
   Placeholder *getPlaceholderByName(llvm::StringRef name) const;
 
-  /// Inserts the Placeholder-Tensor pair.
-  void insert(Placeholder *P, Tensor &&T);
+  /// Inserts the Placeholder-Tensor pair. If \p overwrite is true then
+  /// deletes an existing tensors for this placeholder if it exists.
+  void insert(Placeholder *P, Tensor &&T, bool overwrite = false);
+
+  /// Inserts the Placeholder-Tensor pair. If \p overwrite is true then
+  /// deletes an existing tensors for this placeholder if it exists.
+  void insert(Placeholder *P, std::unique_ptr<Tensor> T,
+              bool overwrite = false);
+
+  /// Remove the Placeholder and its mapped Tensor from the context.
```

Comment:
// and returns the removed tensor. Probably obvious from the signature but for completeness.